app/server/api/credit_transfer_api.py

```diff
@@ -458,7 +458,7 @@ def post(self):
 )
 
 credit_transfer_blueprint.add_url_rule(
-    '/credit_transfer/<int:credit_transfer_id>',
+    '/credit_transfer/<int:credit_transfer_id>/',
```

Comment:
I changed this in [PR 105](https://github.com/teamsempo/SempoBlockchain/pull/105) and don't know why

Reply:
i think it was to make the tests easier to write...

Reply:
v stupid in hindsight

Reply:
I hate how this even matters. Feels like u should be able to just use either.